tlslite/tlsconnection.py

```diff
@@ -1766,15 +1777,13 @@ def _clientFinished(self, premasterSecret, clientRandom, serverRandom,
             yield result
         yield masterSecret
 
-    def _clientGetKeyFromChain(self, certificate, settings, tackExt=None):
-        #Get and check cert chain from the Certificate message
-        cert_chain = certificate.cert_chain
-        if not cert_chain or cert_chain.getNumCerts() == 0:
-            for result in self._sendError(AlertDescription.illegal_parameter,
-                    "Other party sent a Certificate message without "\
-                    "certificates"):
-                yield result
+    def _check_certchain_with_settings(self, cert_chain, settings):
```

Comment:
That seems to be named in different style to what it replaces (_clientGetKeyFromChain vs _check_certchain_with_settings)

Reply:
yes, because the old style is deprecated